名称：
	catch() - 抓出评估错误 (evaluation error)。
语法：
	mixed catch( mixed expr );
	mixed catch { ... };

	混合 catch( 混合 运算式 );
	混合 catch { ... };
用法：
	注意，catch 是一个保留字，而不是一个外部函式。

	评估在 { ... } 里面的程式码或是运算式。如果没有错误，catch() 传回
	0。如果有错，会传回一个字串〈以“*”开头〉。

	也可以使用函式 throw() 以立刻传回除了 0 以外的任何值。

	catch() 有点耗费系统资源，请您别到处使用它。把 catch()
	放在出错时可能会摧毁系统结构 (consistency) 的地方。
参考：
	error, throw, error_handler
作者：
	Tim Hollebeek  Beek@ZorkMUD, Lima Bean, IdeaExchange, and elsewhere
翻译：
	spock@muds.net		2000.May.21.	v22 版
